images/pegleg_Dockerfile.sle_15sp1

```diff
@@ -0,0 +1,64 @@
+#
+# (c) Copyright 2019 SUSE LLC
```

Comment:
don't need to keep tt copy rights here?

Reply:
I will added it but there was no att copy right for other ubuntu dockerfiles.